src/main/kotlin/com/hibob/academy/dao/models.kt

```diff
@@ -1,13 +1,17 @@
 package com.hibob.academy.dao
 
-import java.util.*
+import java.sql.Date
 
 data class Example(val id: Long, val companyId: Long, val data: String)
 
 data class OwnerData(val id: Long, val name: String, val companyId: Long, val employeeId: String)
 
+data class Owner(val name: String, val companyId: Long, val employeeId: String)
+
 data class PetData(val id: Long, var name: String, val type: PetType, val companyId: Long, val dateOfArrival: Date, val ownerId: Long?)
```

Comment:
Why var name?

Reply:
I did it in order to have a function that updated the pets name but Eli explained that i should change it to val and use copy in order to change the name